README.md

```diff
@@ -3,24 +3,62 @@
 
 A CKeditor plugin that makes adding binder enabled pre tags easy.
 
-### Installation
+## Installation
 
 ```
 yarn install
 ```
 
-### Start Dev Server
+## Start Dev Server
 
 ```
 yarn start
 ```
 
-### Build Prod Version
+## Build Prod Version
 
 ```
 yarn build
 ```
 
-### Linter
+## Linter
 
 This repo follows Airbnb Javascript guide, see [here](https://github.com/airbnb/javascript) for more information.
+
+
+## Usage with LibreTexts
+
+CKEditor is a rich text editor which enables the user to write content directly inside web pages or online applications. Through the Binder Plugin, authors of textbooks have the option to include code within in their textbook pages to create a more interactive experience for their students.
+
+
+
+
+Currently the plugin supports Python 3, Julia, R, Octave, and SageMath. Additionally, all languages with the exception of SageMath support syntax highlighting for an easier scripting experience. 
+
+
+
+### Step-by-Step Process to use CKEditor Binder Plugin
+
+
+#### Creating a new page
+Authors have the option to add code to a new page.
+
+![](tutorialVisuals/newPage.gif)
+
+#### Editing a exisiting page
+Additionally, authors can also add code to pages that they have already created. To do so, simply click on the "Edit" button.
+![](tutorialVisuals/editOld.gif)
+
+#### Locating the Plugin and Adding Code
+The Binnder Logo is the last button on the navigation bar of the text editor.  
+
+![](tutorialVisuals/ckePlugin.png)  
+
+#### Add Code
+To add a block of code, select the language, and add code in the text space. Remember to press run, and allow the ouput to fully apear in order for it to properly show up on the webpage
+![](tutorialVisuals/addCode.gif) 
+
+### An Example Page
+Remeber to press save once edits to the page are done, the following is an example page utilizing the Binder Plugin. As seen, authors can use libraries and packages available.
+![](tutorialVisuals/examplePage.gif)
```

Comment:
Somehow the image and the text position is messed up. Can you try to fix it?

Reply:
I'm not entirely sure what you mean here. Do you mean that they positions of the two GIFS look different in the below screenshot?
![image](https://user-images.githubusercontent.com/53124295/78321592-4a59ef80-7521-11ea-9afe-b3e18ac5fbc4.png)